crates/uv-toolchain/src/managed.rs

```diff
@@ -196,6 +196,10 @@ impl InstalledToolchains {
     }
 }
 
+static EXTERNALLY_MANAGED: &str = "[externally-managed]
+Error=This toolchain is managed by uv and should not be modified.
```

Comment:
Should we suggest using a virtual environment? (Have you looked at any of the other externally-managed messages?)

Reply:
I did at first but we suggest using a virtual environment in our own error message it seemed overkill. I guess they could be using `pip` but we don't expose a central `bin` for our managed toolchains so I don't expect them to be on the PATH.

There's a Debian example over at https://packaging.python.org/en/latest/specifications/externally-managed-environments/#marking-an-interpreter-as-using-an-external-package-manager 

I figure we could expand on this with more concrete recommendations if we need to later?